go.mod

```diff
@@ -51,6 +52,21 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
```

Comment:
Does this need a `go mod tidy`? Its peculiar to have more than 2 `require` blocks.

Reply:
I did run go mod tidy and that didn't do anything. I can manually move it into two blocks: one for direct and one for indirect.